src/shareon.js

```diff
@@ -13,7 +13,10 @@ import "./shareon.css";
  *   media?: string,
  *   text?: string,
  *   via?: string,
- *   fbAppId?: string
+ *   fbAppId?: string,
+ *   posttype?: string,
+ *   tags?:[string],
+ *   content?: string || [string]
```

Comment:
Since the type of the post is always a link, this is also not needed, as we can use `url` instead